### PR TITLE
doc: add security policy page

### PR DESCRIPTION
**Pull request by romani**
from https://github.com/checkstyle/checkstyle/issues/8936

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+| Version      | Supported          |
+| ------------ | ------------------ |
+| LATEST only  | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+We have mail-list
+https://checkstyle.org/mailing-lists.html
+you can find admin emails in it.
+you can find my email in my commits, you can clone our repo and see it by "git log".
```
Comment (rnveach) [inline SECURITY.md:17]: "my" seems confusing when reading this page. I doubt users will look up who created this MD.

Why not point them to admins at https://github.com/checkstyle/checkstyle/blob/master/pom.xml#L32 ? We could add our emails there.

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+If we are not responding, please keep pining us, as final ersort create issue at
+https://github.com/checkstyle/checkstyle/issues with note that you have security issue
+
+if the vulnerability is accepted we create issue (without much details) on Github Issue tracker
+and we put label "approved" on it.
+if the vulnerability is declined, we can keep it private
+or if you would like to keep record of it issue tracker, we can do this also.
```
Comment (rnveach) [inline SECURITY.md:25]: `of it issue tracker` => `of it in issue tracker`

`we can do this also` => `you can do this also` ???

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+| LATEST only  | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+We have mail-list
+https://checkstyle.org/mailing-lists.html
+you can find admin emails in it.
+you can find my email in my commits, you can clone our repo and see it by "git log".
+
+If we are not responding, please keep pining us, as final ersort create issue at
```
Comment (rnveach) [inline SECURITY.md:19]: `pining us, as` => `pining us. As`

**Review by rnveach — CHANGES_REQUESTED** [review]
Capitalize first letter in sentence and add periods to end of sentences.

File: SECURITY.md (hunk 3 above)
Comment (strkkk) [reply SECURITY.md:19]: also ersort -> resort

File: SECURITY.md (hunk 1 above)
Comment (romani) [reply SECURITY.md]: sorry for copy-paste from private conversation .... fixed.
Am not sure why but I still have old fashioned position to not print email in plain text.

I updated a bit wording, please take a look.

File: SECURITY.md (hunk 3 above)
Comment (romani) [reply SECURITY.md]: fixed.

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+## Reporting a Vulnerability
+
+List of maintainers: https://checkstyle.org/team.html
+Mail-list for any questions: https://checkstyle.org/mailing-lists.html
+You can find admin emails in mail-list.
+You can find emails of maintainers in commits, you can clone our repo and see it by "git log".
+
+If we are not responding, please keep pining us. As final resort create issue at
+https://github.com/checkstyle/checkstyle/issues with note that you have security issue
```
Comment (rnveach) [inline SECURITY.md:20]: `final resort create issue` => `final resort, create an issue`
`with note that you have security issue` => `with a note that you have a security issue.`

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+If the vulnerability is accepted we create issue (without much details) on Github Issue tracker
```
Comment (rnveach) [inline SECURITY.md:22]: `is accepted we create issue` => `is accepted, we will create an issue`

**Review by rnveach — CHANGES_REQUESTED** [review]
Last changes.

Have you run this against @pbludov MD style checker?

File: SECURITY.md (hunk 5 above)
Comment (romani) [reply SECURITY.md]: fixed.

File: SECURITY.md (hunk 4 above)
Comment (romani) [reply SECURITY.md]: fixed.

**Comment by romani** [comment]
> Have you run this against @pbludov MD style checker?

I can not do this on local, as I do not want to install ruby on my laptop. I rebased this PR, CI will show what to fix.

**Review by rnveach — APPROVED** [review]

**Pull request merged**